packages/synapse-react-client/src/lib/style/components/facet_nav/_facet-nav-panel.scss

```diff
@@ -119,10 +119,11 @@ $facetNavPanel-btn-margin: 0 1px;
   margin: auto;
   .SaveFiltersButton {
     display: block;
-    text-transform: uppercase;
     margin: auto;
     width: 250px;
-    padding: 5px 10px;
+    min-width: 150px;
+    min-height: 20px;
+    padding: 5px;
     font-size: 16px;
   }
```

Comment:
Moved styling, see these comments: [one](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/126/files#r1166127561) and [two](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/126/files#r1166129945)